### ISSUE TITLE
Statement conflict tests putting full path in statementId param

### ISSUE THREAD

**Issue by milt** [adlnet/xAPI_LRS_Test#40]
https://github.com/adlnet/xAPI_LRS_Test/blob/master/src/steps/conflict.js#L18

The statement's id is set properly directly below, with `this.featureResource.conflictDir.slice(this.featureResource.conflictDir.lastIndexOf("/") + 1)`

**Comment by brianjmiller** [comment]
Yeah, we found this one as well, the larger issue is that it revealed that all the statement conflict tests have a false positive in them since it should be causing a failure of the test. We have a PR nearly ready to submit that fixes the basic problem you identified and the larger issue in general. We'll submit it as soon as I land my PR for the items raised in #13. Both PRs should be landing very soon.

**Comment by milt** [comment]
Cool! Figured it was known.
